openshift/rh-che.config.yaml

```diff
@@ -32,6 +32,7 @@ data:
   che-openshift-precreate-subpaths: "false"
   che-openshift-secure-routes: "true"
   che.predefined.stacks.reload_on_start: "true"
+  che-system-admin-name: "admin"
```

Comment:
Do we have a user with this name already?

Reply:
I believe it would be applied via cm "on dsaas / dsaas-stg"

Reply:
yeah, you are right. It doesn't have to match this one

Reply:
Is it a problem, if such user does not exist? If so, it could be a problem for PR check deployment.

Reply:
no, I believe there should not be any problems if the user does not exist - no admin user is going to be assigned.

Reply:
no, but I set it to be matching [this](https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties#L27)